Jenkinsfile-sbom-release

```diff
@@ -0,0 +1,195 @@
+@Library(['private-pipeline-library', 'jenkins-shared']) _
+
+import com.sonatype.jenkins.pipeline.OsTools
+
+import groovy.json.JsonSlurper
+
+IQ_URL_BASE = "https://sonatype.sonatype.app/platform"
+REPO_BASE_URL = "https://repo.sonatype.com/service/rest"
+TARGET_REPO_NAME = "sonatype-sboms"
+REDHAT_SBOM_REPO_URL_BASE = "https://access.redhat.com/security/data/sbom/beta"
+REDHAT_CONTAINER_API_URL_BASE = "https://catalog.redhat.com/api/containers/v1"
+CYCLONEDX_VERSION = "1.5"
+
+properties([
+    parameters([
+        string(name: 'docker_nexus3_tag', defaultValue: '',
+            description: 'Nexus Repository Manager Version. The result SBOMs will be tagged with this version.')
+    ])
+])
+
+
+node('ubuntu-zion') {
```

Comment:
The use of scripted pipelines is not encouraged here at Sonatype.  Please see https://sonatype.atlassian.net/wiki/spaces/CDI/pages/10393538/Jenkins+Best+Practices

This should probably use the [dockerizedRunPipeline](https://sonatype.atlassian.net/wiki/spaces/CDI/pages/10392908/Dockerized+Run+Pipeline) especially since it uses Python which is not version pinned on our agents.

Reply:
Resolved in [db05880](https://github.com/sonatype/docker-nexus3/pull/191/commits/db05880884bda56fc3988f0b9a12790b4aa16afb)